### PR TITLE
Lesson-1-4

### PR DESCRIPTION
**Pull request by leipei92**
Homework Week 1-4.

### CODE REVIEW

File: src/App.js
Comment (JoHeyJo) [inline src/App.js:10]: Great job implementing the spread operator!

File: src/App.js
Comment (JoHeyJo) [inline src/App.js]: This `addTodo` is not doing anything for you except triggering an interesting error in your console. See if you can spot it. What are your guesses as to what you should do? Also, double-check the instructions.

Let's try collaborating through github for this issue. Sometimes this is quicker and much easier than setting up a meeting. You can respond directly in this thread while we figure it out.

Always happy to meet for a call but this is a necessary workflow technique that you need to get comfortable with in a professional environment.

File: src/AddTodoForm.js
Comment (JoHeyJo) [inline src/AddTodoForm.js:18]: Fantastic. Good job nailing this!

File: src/TodoListItem.js
Comment (JoHeyJo) [inline src/TodoListItem.js]: Nice job destructuring here but the variable name is a little misleading. We aren't passing the whole todo object. It's just one specific part of the object now.

**Review by JoHeyJo — CHANGES_REQUESTED** [review]
Really nice job getting your app to work and applying the destructuring technique and spread operator!

There is one thing that needs to be addressed before approval and that is the `addTodo` in your `App` component.

File: src/App.js
Comment (leipei92) [reply src/App.js]: After 1:1 with Sergey, I removed this section, but  didn't push the changes.

File: src/TodoListItem.js
Comment (leipei92) [reply src/TodoListItem.js]: I am confused. If i change the variable name then it won't display the value of the input after submission. It just shows an empty value. Am I missing something here?

Comment (JoHeyJo) [reply src/TodoListItem.js]: You would need to change the variable name where you're passing the variable as well.

**Pull request merged**
